README.md

```diff
@@ -27,6 +36,7 @@ Apply the following secret, filling in your own account details based on the env
     "type": "Opaque"
 }
 ```
+> **Note:** If you find yourself changing the secret while the shredder is running, you'll need to kill the shredder pod to ensure the updated secret is pulled when the new pod is created.
```

Comment:
You could explain that this happens because the shredder reads the secret at runtime and stores it in memory so changes won't be applied. Not important tho